backends-velox/src/main/scala/io/glutenproject/execution/RowToVeloxColumnarExec.scala

```diff
@@ -58,132 +60,150 @@ case class RowToVeloxColumnarExec(child: SparkPlan) extends RowToColumnarExecBas
     val localSchema = schema
     child.execute().mapPartitions {
       rowIterator =>
-        if (rowIterator.isEmpty) {
```

Comment:
move the code to `RowToVeloxColumnarExec.toColumnarBatchIterator ` so that columnar serializer can reuse it